src/Room/Room.js

```diff
@@ -1,14 +1,58 @@
 import React, { useState, useEffect } from "react";
 import { StyleSheet, View, Text, Image, TouchableOpacity } from "react-native";
 import { getRoom, getMemories } from "../apiCalls";
-import  Memory  from "../Memory/Memory";
+import Memory from "../Memory/Memory";
 
 export default function Room(props) {
   const selectedRoom = props.route.params.id;
 
   const [room, setRoom] = useState({});
   const [memories, setMemories] = useState([]);
-   const [memoryIsSelected, setMemoryAsSelected] = useState(false);
+  const [selectedMemory, setSelectedMemory] = useState(null);
+
+  const createMemories = () => {
+    if (memories.length > 0) {
+      let memoryComponents = memories.map((memory) => {
+        return (
+          <TouchableOpacity
+            key={memory.id}
+            style={memoryStyles[memory.id]}
+            onPress={() => {
+              setSelectedMemory(memory.id);
+            }}>
+            {selectedMemory === memory.id && (
+              <Memory memory={memory} setSelectedMemory={setSelectedMemory} />
+            )}
+          </TouchableOpacity>
+        );
+      });
+      return memoryComponents;
+    }
+  };
+
+  const createMemoryStyles = () => {
+    if (memories.length <= 0) {
+      return;
+    } else {
+      let memoryCoordinates = memories.reduce((memoryCoordinates, memory) => {
+        if (!memoryCoordinates[memory.id]) {
+          memoryCoordinates[memory.id] = {
+            zIndex: 2,
+            position: "absolute",
+            borderRadius: 75,
+            borderWidth: 4,
+            borderColor: "red",
+            top: memory.y - 50,
```

Comment:
We should potentially check to see how this is rendered on a bigger/smaller tablet. Just curious if it still works.